.circleci/config.yml

```diff
@@ -66,14 +66,20 @@ commands:
           name: Gather test results
           when: always
           command: |
-            FILES=`find . -name reports`
+            FILES=`find . -name reports -not -path './build/reports'`
```

Comment:
The -not -path excludes the build/reports directory otherwise we copy part of license report (not the whole report just the sub sub directories with the metadata info).